stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -1167,6 +1167,15 @@ public func runAllTests() {
   }
 }
 
+#if SWIFT_RUNTIME_ENABLE_LEAK_CHECKER
+
+@_silgen_name("swift_leaks_startTrackingObjects")
+func startTrackingObjects(_: UnsafeMutablePointer<Void>)
```

Comment:
@gribozavr This is cargo-culted from the benchmark code. When I wrote it, I probably just didn't know about UnsafePointer<CChar>. It should be fine to change this.